fast_td3/fast_td3/train_v2.py

```
@@ -0,0 +1,868 @@
+            raise ValueError("No GPU available")
+    print(f"Using device: {device}")
+
+    env_type = "humanoid_bench"
+    envs = HumanoidBenchEnv(args.env_name, args.num_envs, device=device)
+    eval_envs = envs
+    render_env = HumanoidBenchEnv(
+        args.env_name, 1, render_mode="rgb_array", device=device
+    )
+
+    n_act = envs.num_actions
+    n_obs = envs.num_obs if type(envs.num_obs) == int else envs.num_obs[0]
```

Ruff (0.14.5)

Use `is` and `is not` for type comparisons, or `isinstance()` for isinstance checks

(E721)